civihr_default_theme/scss/civihr_default_theme.style.scss

```diff
@@ -117,6 +117,7 @@
 @import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/overrides/style/tables";
 @import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/leave-report";
 @import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/leave-calendar";
+@import "SCSSROOT/uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/manager_calendar";
```

Comment:
I think `manager_calendar` is still fine because still it shows the calendar for the managees. Admin is getting all contacts because he manages all.